x-pack/plugins/actions/server/create_execute_function.ts

```diff
@@ -72,13 +79,23 @@ export function createExecuteFunction({
     });
 
     await taskManager.schedule({
-      taskType: `actions:${actionSavedObject.attributes.actionTypeId}`,
+      taskType: `actions:${actionTypeId}`,
       params: {
         spaceId,
         actionTaskParamsId: actionTaskParamsRecord.id,
       },
       state: {},
       scope: ['actions'],
     });
+
+    async function getActionTypeId(actionId: string): Promise<string> {
```

Comment:
As a side note, we might want to look at denormalizing the action type along side action id, where appropriate, to avoid this SO lookup **just to get the action type**.  OTOH, it's only when actions are executed, so ... it's not on every turn of an alert ...

Reply:
(Moment not writing a comment haunts me): I think the lookup is also there to ensure the user has access to read that action, otherwise it would fail the attempt to create a task executing that action.

Sigh note, this would also mean pre-configured actions don't work with feature controls at this time.

Reply:
Created issue here: #63496.